### PR TITLE
[MINOR] Fix typos and wrong package name

### PR DESCRIPTION
**Pull request by turboFei**
### What changes were proposed in this pull request?

Fix some typos.

### Why are the changes needed?

Ditto.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Not needed.

### CODE REVIEW

**Comment by turboFei** [comment]
cc @pan3793

File: master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/SingleMasterMetaManager.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public SingleMasterMetaManager(
     this.initialEstimatedPartitionSize = conf.initialEstimatedPartitionSize();
     this.estimatedPartitionSize = initialEstimatedPartitionSize;
     this.appDiskUsageMetric = new AppDiskUsageMetric(conf);
-    this.rackResolver = new CelebornRackResolver(conf);
```
Comment (pan3793) [inline master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/SingleMasterMetaManager.java:50]: this actually is a bug

Comment (turboFei) [reply master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/SingleMasterMetaManager.java:50]: ok, will raise another pr.

Comment (pan3793) [reply master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/SingleMasterMetaManager.java:50]: move this change to a dedicated patch

**Review by pan3793 — APPROVED** [review]

**Pull request closed without merging**